### ISSUE TITLE
Database table update problem

### ISSUE THREAD

**Issue by akelyasir** [photino/zino#42]
Hi @photino 

There is no problem in adding to the database, but there are problems in updating operations. For example, adding a new user is successful. But when I want to update the user it fails. 

First mistake:
`"detail": "no column found for name: access_key_id",`

When I replace the projection data with * in the try_get_model function. The column exists, but this time the role column gives an error.

`"detail": "error occurred while decoding column \"roles\": expected value at line 1 column 1: expected value at line 1 column 1",`

I couldn't find the exact problem.

**Comment by photino** [comment]
Have you considered dropping the table and recreating it by the ORM?

**Comment by akelyasir** [comment]
@photino 

Yes I rebuilt it and it is just by ORM nature. I'm still getting the same errors without making any changes to your latest version.

**Comment by photino** [comment]
The error `"detail": "no column found for name: access_key_id"` is due to the fact that the `access_key_id` field has been marked as `not_null, writeonly`. If you remove either of them, it should work again. I will fix this compatibility in the next commit.

**Comment by akelyasir** [comment]
Thanks @photino. When I do it that way, I get the error below. Can you check it too?

`"detail": "error occurred while decoding column \"roles\": expected value at line 1 column 1: expected value at line 1 column 1",`

**Comment by akelyasir** [comment]
@photino Let me write down the steps I did in order for it to be understood more clearly.

1- I downloaded the last commit.
2- I made database settings in your sample application actix-web.
3- Values that I send to /user/new with Postman.
`{
    "name": "New User",
    "account": "account",
    "password": "Password123",
    "email": "usermail",
    "roles": ["admin"]
}`

4- I tried to update for the created user. Here are a few body I've posted examples of.

`{
    "name": "Updated Name",
    "email": "mail@email.com",
    "description": "aasdasd",
}`

`{
    "name": "Updated Name",
    "email": "mail@email.com",
    "description": "aasdasd",
    "roles": ["admin:user", "auditor"]
}`

A few updates ago, these processes were not giving any errors. I am using **postgresql** as database for **Actix**. After setting up **mysql 8.0** database for **axum** it gives a different error on user update after new user actions. 

A few updates ago, access_key_id, roles, account etc. were not required and actix postgresql operations were not giving any errors and operations were successful. The roles column in postgresql gives an error whether we enter the mandatory values or disable them. Id column gives error in mysql.
